[2024-06-23T23:41:39.456+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-23T23:41:39.480+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nasa_to_s3.fetch_nasa_data manual__2024-06-23T23:41:34.508993+00:00 [queued]>
[2024-06-23T23:41:39.487+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nasa_to_s3.fetch_nasa_data manual__2024-06-23T23:41:34.508993+00:00 [queued]>
[2024-06-23T23:41:39.488+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-23T23:41:39.499+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_nasa_data> on 2024-06-23 23:41:34.508993+00:00
[2024-06-23T23:41:39.511+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1350) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-23T23:41:39.513+0000] {standard_task_runner.py:63} INFO - Started process 1352 to run task
[2024-06-23T23:41:39.513+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nasa_to_s3', 'fetch_nasa_data', 'manual__2024-06-23T23:41:34.508993+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/extractAPIdata.py', '--cfg-path', '/tmp/tmp5jxs_ozj']
[2024-06-23T23:41:39.515+0000] {standard_task_runner.py:91} INFO - Job 19: Subtask fetch_nasa_data
[2024-06-23T23:41:39.556+0000] {task_command.py:426} INFO - Running <TaskInstance: nasa_to_s3.fetch_nasa_data manual__2024-06-23T23:41:34.508993+00:00 [running]> on host e0fa33b4439a
[2024-06-23T23:41:39.632+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='nasa_to_s3' AIRFLOW_CTX_TASK_ID='fetch_nasa_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-23T23:41:34.508993+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-23T23:41:34.508993+00:00'
[2024-06-23T23:41:39.634+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-23T23:41:40.883+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-23T23:41:40.884+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-23T23:41:40.893+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=nasa_to_s3, task_id=fetch_nasa_data, run_id=manual__2024-06-23T23:41:34.508993+00:00, execution_date=20240623T234134, start_date=20240623T234139, end_date=20240623T234140
[2024-06-23T23:41:40.930+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-23T23:41:40.954+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-23T23:41:40.956+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
